xbmc/games/dialogs/osd/DialogGameVideoSelect.cpp

```diff
@@ -29,8 +29,8 @@ using namespace KODI;
 using namespace GAME;
 
 #define CONTROL_HEADING 1
-#define CONTROL_THUMBS 11
-#define CONTROL_DESCRIPTION 12
+#define CONTROL_THUMBS 10811
```

Comment:
Would be nice to update these defines to constexpr and use an anonymous namespace but that can be done in the future.

Reply:
Will do in this PR